pom.xml

```diff
@@ -340,6 +341,9 @@
         <plugin>
           <artifactId>maven-javadoc-plugin</artifactId>
           <version>2.10.1</version>
+          <configuration>
+            <quiet>${javadoc.quiet}</quiet>
```

Comment:
why not either just define the standard property `quiet` to have a default value of `true` or just force it true and doc how to turn it back. You are breaking the javadoc plugin's docs by introducing a new property.

We seem to go property mad to avoid people having to "learn Maven" and all we do is raise the bar for people who want to learn it!